### ISSUE TITLE
Weird dropdown bug

### ISSUE THREAD

**Issue by Wintekso** [rainlab/builder-plugin#108]
Hello,

I have a weird issue with builder when creating a new plugin: 

When selecting the plugin icon the list appears behind the dropdown box? 

![capture](https://cloud.githubusercontent.com/assets/19787128/17988473/706a8b00-6b1e-11e6-812c-e744101748bb.PNG)

Is this a bug or is it something else that could be causing this? 

Many thanks

Samuel

**Comment by daftspunk** [comment]
Couldn't replicate this on my end.  What browser and OS are you using?

Using Chrome 53 on Windows 10

**Comment by bhargavy** [comment]
Hello,

Im getting a similar issue in on Chrome 53 and Firefox 48 in Windows 8.

**Comment by daftspunk** [comment]
Also confirm are you using a touch-enabled device? eg: Laptop with touch screen.

**Comment by Wintekso** [comment]
No have reproduced on multiple devices.

Many thanks

Samuel Winter
Winter Solutions Ltd


-------- Original message --------

**Comment by daftspunk** [comment]
This appears to be a duplicate of https://github.com/octobercms/october/issues/1328

Let's move the investigation to this issue.

**Comment by axomat** [comment]
I have reproduced this (or at least the duplicate issue because I am not convinced these are the same issues) with a fresh october installation with just the demo and test plugins installed.

If I go to CMS / add new page, "layout" is showing a simple dropdown not the select2 dropdown that you would normally expect.

The page source says "The select2('focus') method was called on an element that is not using Select2." as an error. This error occurs for every dropdown in the backend,

![image](https://cloud.githubusercontent.com/assets/6474911/18955448/70586e2e-864f-11e6-8340-9f0cb9c8a1af.png)

It only happens on Chromium 52. The previous poster on the original issue suggested that it was related to a new JS engine in 52 but I dont know if that is the case.

I have tried to research this issue for hours to help resolve it but I really dont understand enough to fix it. I realise it is difficult to resolve if you cant reproduce it and to be honest I have moved to FIrefox to get around it. 

It may also be relevant that I use a touch screen computer and have always had a bug in October that does not allow the backend top bar to be scrolled left and right with the mouse (in my dev box I have loads of plugins showing). I live with that problem because it is a minor use case but I mention it in case its relevant to this issue.

Sorry I cant be more helpful.
